html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form id = 'coding-form'>
+     <div>Email:</div>
+     <label for="email">
+      <input type="email" id="email" name="user_email">
+     </label>
+
+     <div>Why is coding cool?</div>
+     <label for="reason">
+      <select id="reason">
+        <option value='1'>Select an option</option>
+        <option value='2'>Because it's interesting</option>
+        <option value='3'>Because it's fun</option>
+        <option value='4'>Because it's powerful</option>
+      </select>
+     </label>
```

Comment:
`label` needs to have some text associated with it. Utilize your text content inside your `div` above this